packages/react-components/src/components/SendBox.test.tsx

```diff
@@ -6,6 +6,12 @@ import { SendBox } from './SendBox';
 import { renderWithLocalization, createTestLocale } from './utils/testUtils';
 import { screen } from '@testing-library/react';
 import { registerIcons } from '@fluentui/react';
+// /* @conditional-compile-remove(mention) */

```

Comment:
Would this still work if the conditional compile comment is commented out?

Reply:
Goot catch! It looks like it works but I'll fix it